test/parallel/test-stream-auto-destroy.js

```diff
@@ -86,27 +86,29 @@ const assert = require('assert');
 {
   const r = new stream.Readable({
     read() {
-      r2.emit('error', new Error('fail'));
+      r2.destroy(new Error('fail'));
```

Comment:
Please not this change. Calling `emit('error'` is not something that should be done. But do we need to support it?